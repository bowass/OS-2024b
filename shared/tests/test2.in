echo hello world!
ls sosdoskls &
sleep 1
date hi nice exit
history &
exit
8888888888888888888888
