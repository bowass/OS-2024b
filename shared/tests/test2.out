hello world!
ls: cannot access 'sosdoskls': No such file or directory
date: extra operand ‘nice’
Try 'date --help' for more information.
my-shell> my-shell> my-shell> my-shell> my-shell> 5 	 history &
4 	 date hi nice exit
3 	 sleep 1
2 	 ls sosdoskls &
1 	 echo hello world!
my-shell> 